src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -259,6 +259,12 @@ opcode(struct sched_cmd *cmd)
 	return cmd->packet->opcode;
 }
 
+static inline u32
+type(struct sched_cmd *cmd)
```

Comment:
type is very close to some keywords, would some function name like cmd_type, get_cmd_type more readable?